openmdao_book/other_useful_docs/developer_docs/doc_style_guide.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Is the bullet format for</p><p> </p><ul><li>Phrase or sentence ended by a period.</li><li>No empty space between the text and the triple double quotes.</li></ul><p>Correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>